src/MessageReader.ts

```diff
@@ -104,6 +106,10 @@ export class MessageReader<T = unknown> {
   }
 }
 
+function isConstantModule(def: MessageDefinition): boolean {
+  return def.definitions.length > 0 && def.definitions.every((field) => field.isConstant);
+}
```

Comment:
What would it mean for a `module` to have a non-constant field? Is there such a thing, or would it have to be in a `struct`? I guess my question is, why are we checking for constant fields as opposed to checking whether it is a `struct {}` or a `module {}`?